test/tools/yulopti.cpp

```diff
@@ -127,6 +141,19 @@ class YulOpti
 			sortedOptions.push_back(optionAndDescription);
 		}
 
+		sort(
```

Comment:
Does ranges-v3 have a sort option like python where you do not specify the comparator but instead a transformation function that returns a key and the comparison function for the key is used? Then this could be a bit shorter, I think.

Reply:
I don't know but it would prefer that it it does :) I'll check.

Reply:
Done. Actually, it was a good chance to get more familiar with ranges-v3. I ended up rewriting it basically from scratch but I learned something new so I think it was worth it. `ranges::views::stride()` also simplified the calculation a bit.